src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -4110,6 +4110,14 @@ DWORD ProfToEEInterfaceImpl::GetModuleFlags(Module * pModule)
 
         // Intentionally not checking for flat, since NGEN PEImages never have flat
         // layouts.
+
+#ifdef FEATURE_READYTORUN
+        PEImageLayout * pPEImageLayout = pPEFile->GetLoaded();
+        if (pPEImageLayout != nullptr && pPEImageLayout->HasReadyToRunHeader())
```

Comment:
Having a R2R header means the image is capable of being used as R2R, but it is also possible to ignore that portion of the image and treat the image as a regular IL image. Depending on what the flag is trying to signify you might want pModule->IsReadyToRun() instead.